### PR TITLE
Adding verification to packageTask

### PR DESCRIPTION
**Pull request by FernandoMorais**
Verification added to packageTask to avoid errors in execution for tasks that don't have common dependencies.

### CODE REVIEW

File: package.js [hunk 1]
```diff
@@ -274,15 +274,17 @@ function packageTask(pkgPath, commonDeps, commonSrc){
...
-                var taskDeps;
-                if ((taskDeps = commonDeps[task.name])) {
-                    taskDeps.forEach(function (dep) {
-                        gutil.log('Linking ' + dep.module + ' into ' + task.name);
-                        var src = path.join(commonSrc, dep.module);
-                        var dest = path.join(tgtPath, dep.dest);
-                        shell.mkdir('-p', dest);
-                        shell.cp('-R', src, dest);
-                    })
+                if(commonDeps){
```
Comment (ericsciple) [inline package.js]: Why this change? If the task doesn't have a common dependency then `commonDeps[task.name]` will be null. `commonDeps` is never null. The entire object is loaded once in the main script and passed into the function.

Comment (FernandoMorais) [inline package.js]: Hi @ericsciple I went in error when calling pkgm.PackageTask() without commonDeps param on a custom gulp build task. I think that, since the param is optinal, would be better check to avoid errors (just to keep things SOLID).

Comment (ericsciple) [inline package.js]: Oh I see now. You hit the problem in a *different* repo which is using some of the build scripts from this repo.

Comment (FernandoMorais) [inline package.js]: That's it! :)

**Comment by kmkumaran** [comment]
@bryanmacfarlane can you accept/close as appropriate.

**Comment by bryanmacfarlane** [comment]
closing since we're going to create shared modules for others to build tasks ....
